src/main/java/com/twilio/rest/accounts/v1/credential/AwsCreator.java

```diff
@@ -89,6 +89,7 @@ public Aws create(final TwilioRestClient client) {
                 restException.getCode(),
                 restException.getMoreInfo(),
                 restException.getStatus(),
+                restException.getDetails(),
```

Comment:
Can't we just pass `restException` into an `ApiException` constructor? Then future changes won't update so many files.

Reply:
yeah thats a good idea, let me do that instead

Reply:
@childish-sambino removed the old constructor and added a new one with restException